cloudslang-content-verifier/src/test/java/io/cloudslang/lang/tools/build/validation/StaticValidatorTest.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * (c) Copyright 2016 Hewlett-Packard Enterprise Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package io.cloudslang.lang.tools.build.validation;
+
+import io.cloudslang.lang.compiler.modeller.model.Flow;
+import io.cloudslang.lang.compiler.modeller.model.Metadata;
+import io.cloudslang.lang.entities.bindings.Input;
+import io.cloudslang.lang.entities.bindings.Output;
+import io.cloudslang.lang.entities.bindings.Result;
+import io.cloudslang.lang.entities.bindings.ScriptFunction;
+import io.cloudslang.lang.entities.bindings.values.ValueFactory;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.io.File;
+import java.net.URISyntaxException;
+import java.util.*;
+
+/**
+ * Created by bancl on 8/31/2016.
+ */
+public class StaticValidatorTest {
+
+    private static final Set<String> SYSTEM_PROPERTY_DEPENDENCIES = Collections.emptySet();
+
+    private StaticValidator staticValidator;
+
+    @Rule
+    public ExpectedException exception = ExpectedException.none();
+
+    @Before
+    public void setup() {
+        staticValidator = new StaticValidatorImpl();
+    }
+
+    @Test
+    public void missingDescriptionForInput() throws URISyntaxException {
+        List<Input> inputList = new ArrayList<>();
+        inputList.add(new Input.InputBuilder("input1", "value1").build());
```

Comment:
this can be written much less verbose using `Lists.netArrayList` from guava and just comma